docs/release_notes/v1.0.0-rc.1.md

```diff
@@ -10,7 +10,7 @@ This is a release candidate version as we progress towards Dapr v1.0.0. We appre
 See [this](#upgrading-to-dapr-100-rc1) section on upgrading Dapr to version 1.0.0-rc.1.
 
 
-> Dapr CLI version 1.0.0-rc.1 is not compatible with previous runtime versions. Download Dapr CLI 1.0.0-rc.1 from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+> Dapr CLI version 1.0.0-rc.1 is not compatible with previous runtime versions. Download the Dapr CLI 1.0.0-rc.1 from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH, or use the rc1 install script in the Getting Started guide.
```

Comment:
@msfussell  isn't this CLI version 1.0.0-rc.2?

Reply:
yes